definitions/checks/pulpcore/group_ownership_check.rb

```diff
@@ -8,10 +8,7 @@ class GroupOwnershipCheck < ForemanMaintain::Check
       end
 
       def run
-        group_id = File.stat('/var/lib/pulp/content/').gid
-        if Etc.getgrgid(group_id).name != 'pulp'
-          fail! "Please run 'foreman-maintain prep-6.10-upgrade' prior to upgrading."
-        end
+        File.stat('/var/lib/pulp/content/').gid
```

Comment:
This check now does nothing (besides fetching the `gid` of `/var/lib/pulp/content` and never using it for anything). I'd argue the whole check can go away now?

Reply:
(also, it's `manual_detection` and the last use was in the already removed `definitions/scenarios/upgrade_to_satellite_6_10.rb`)

Reply:
You aint got to tell me twice to delete code, even though you tried.